shopinvader_customer_price/models/shopinvader_backend.py

```diff
@@ -44,21 +44,10 @@ def _get_partner_pricelist(self, partner):
             pricelist = partner.property_product_pricelist
         return pricelist
 
-    @tools.ormcache("partner.id", "self.company_id.id")
```

Comment:
any reason to get rid of this? The reason why we get 2 methods is that you can cache only the ID safely. 
And since this func is called for each product in the list that we send back, it's handy to cache it.

Reply:
In v14 get_fiscal_position returns a recordset and not an id: https://github.com/odoo/odoo/commit/81245528d18723845f80a5e4d348d044b730b655

Reply:
But it may be better for perfs if I return `fpos.id` in the cached function and the browse it in the other one. What would you prefer ?

Reply:
the main issue is that this method will be called many times and I'd prefer to avoid to retrieve the fiscal position when not necessary (eg: during the same rest call). So, yes, I would return .id and browse it later again.

Reply:
Ok I've done it